build/scripts/linting.mk

```diff
@@ -85,7 +85,7 @@ nilaway:
 	for dir in $$dirs; do \
 		count=$$((count + 1)); \
 		printf "[%d/%d modules complete] Running nilaway in %s\n" $$count $$total $$dir && \
-		(cd $$dir && go run go.uber.org/nilaway/cmd/nilaway -exclude-errors-in-files "pkg/components/module,pkg/deposit,pkg/cosmos" -v ./...) || exit 1; \
+		(cd $$dir && go run go.uber.org/nilaway/cmd/nilaway -exclude-errors-in-files "pkg/components/module,pkg/deposit,pkg/cosmos,pkg/eip4788" -v ./...) || exit 1; \
```

Comment:
reason?

Reply:
Another bindings package similar to "pkg/deposit"